CI/setter.sh

```diff
@@ -1,20 +1,22 @@
-cd ..
+# installing python test libraries
 python -m pip install --upgrade pip
-pip install flake8==4.0.1
-pip install pytest==7.1.0
-pip install pydantic==1.9.1
-pip install testcontainers==3.6.0
 if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
 
+# download <swagger.yaml> file
+cd /home/runner/work/lightning_py/lightning_py
 curl -O https://raw.githubusercontent.com/aerokube/selenium-openapi/master/selenium.yaml
 
+# install swagger codegen
 git clone https://github.com/swagger-api/swagger-codegen
 cd swagger-codegen
 git checkout 3.0.0
 mvn clean package
 java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate \
-   -i swagger.yaml \
+   -i /home/runner/work/lightning_py/lightning_py/selenium.yaml \
    -l python \
-   -o /python
+   -o /home/runner/work/lightning_py/lightning_py/python
+cd /home/runner/work/lightning_py/lightning_py
 
-cd python/ && python setup.py install && cd ..
+# setup swagger API
+cd /home/runner/work/lightning_py/lightning_py/python && python setup.py install && cd ..
```

Comment:
Default working directory is `/home/runner/work/lightning_py/lightning_py`, so can just do `pushd python && python setup.py && popd`.